.github/workflows/common.yml

```diff
@@ -28,7 +28,14 @@ jobs:
   unitTest:
     name: Unit test
     runs-on: ubuntu-18.04
+    services:
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 5672:5672
```

Comment:
why do we need this docker? we can use `dockertest`. See https://github.com/hyperledger/aries-framework-go-ext/blob/main/component/storage/mysql/store_test.go#L37

Reply:
Is `dockertest` a required approach?  The mongodb PR was approved and merged using this approach.

Reply:
I can't say that `dockertest ` is required. But it is better for sure. BTW `mongodb` tests don’t work. I will create an issue for that.

Reply:
https://github.com/hyperledger/aries-framework-go-ext/issues/22

Reply:
FYI: https://github.com/hyperledger/aries-framework-go-ext/issues/22

Reply:
I'll make the switch to both today, but I'm curious why `dockertest` is better.

Reply:
@pfeairheller `dockertest` simplifies the usage for users. You don't need to run or kill containers. Tests can be run easily.

Reply:
Ah, got it.  Makes it easier to run locally.  Makes sense, thanks for the answer.

Reply:
Done.